static/app/components/performance/spanSearchQueryBuilder.tsx

```diff
@@ -40,6 +84,19 @@ export function SpanSearchQueryBuilder({
   const api = useApi();
   const organization = useOrganization();
   const {selection} = usePageFilters();
+  const [groupBys] = useGroupBys();
+  const [visualizes] = useVisualizes();
+  const [resultsMode] = useResultMode();
```

Comment:
You cannot use these hooks in this component directly as this is a generic component shared by other pages while these hooks were made specifically for the explore page.

Reply:
Right @Zylphrex, I'll just pass them down as props.